packages/11ty/_includes/components/page-header.webc

```diff
@@ -0,0 +1,3 @@
+<script webc:type="js">
+this.pageHeader(pageData.data);
```

Comment:
@mphstudios I was trying to render just the `page-header.js` component using a liquid tag, but implementing this WebC wrapper component ended up actually working.

I was messing with this, but it didn't end up working (see https://www.11ty.dev/docs/languages/webc/#using-template-syntax-to-generate-content)
```liquid
<template webc:type="11ty" 11ty:type="liquid,md">
{% pageHeader
  byline_format=byline_format,
  image=image,
  label=label,
  pageContributors=pageContributors,
  subtitle=subtitle,
  title=title
%}
</template>
```